<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1223</width>
    <height>827</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Main window</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0" colspan="2">
     <widget class="QGroupBox" name="SearchEngine">
      <property name="minimumSize">
       <size>
        <width>1187</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>250</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="title">
       <string/>
      </property>
      <widget class="QPushButton" name="Health_B">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>30</y>
         <width>131</width>
         <height>41</height>
        </rect>
       </property>
       <property name="text">
        <string>Health</string>
       </property>
       <property name="checkable">
        <bool>false</bool>
       </property>
       <property name="autoDefault">
        <bool>false</bool>
       </property>
      </widget>
      <widget class="QPushButton" name="Cost_B">
       <property name="geometry">
        <rect>
         <x>180</x>
         <y>30</y>
         <width>131</width>
         <height>41</height>
        </rect>
       </property>
       <property name="text">
        <string>Cost</string>
       </property>
       <property name="checkable">
        <bool>false</bool>
       </property>
       <property name="checked">
        <bool>false</bool>
       </property>
       <property name="autoDefault">
        <bool>false</bool>
       </property>
      </widget>
      <widget class="QPushButton" name="Attack_B">
       <property name="geometry">
        <rect>
         <x>320</x>
         <y>30</y>
         <width>141</width>
         <height>41</height>
        </rect>
       </property>
       <property name="text">
        <string>Attack</string>
       </property>
      </widget>
      <widget class="QPushButton" name="Type_B">
       <property name="geometry">
        <rect>
         <x>470</x>
         <y>30</y>
         <width>131</width>
         <height>41</height>
        </rect>
       </property>
       <property name="text">
        <string>Type</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="Name_T">
       <property name="geometry">
        <rect>
         <x>180</x>
         <y>80</y>
         <width>261</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="placeholderText">
        <string>Nazwa</string>
       </property>
      </widget>
      <widget class="QPushButton" name="Rarity_B">
       <property name="geometry">
        <rect>
         <x>45</x>
         <y>77</y>
         <width>121</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>Rarity</string>
       </property>
      </widget>
      <widget class="QPushButton" name="Region_B">
       <property name="geometry">
        <rect>
         <x>480</x>
         <y>80</y>
         <width>121</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>Region</string>
       </property>
      </widget>
      <widget class="QPushButton" name="Search_B">
       <property name="geometry">
        <rect>
         <x>770</x>
         <y>70</y>
         <width>231</width>
         <height>41</height>
        </rect>
       </property>
       <property name="text">
        <string>Search</string>
       </property>
      </widget>
      <widget class="QComboBox" name="Sort_DB">
       <property name="geometry">
        <rect>
         <x>770</x>
         <y>30</y>
         <width>231</width>
         <height>28</height>
        </rect>
       </property>
       <item>
        <property name="text">
         <string>Alphabetically A-Z</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Alphabetically Z-A</string>
        </property>
       </item>
      </widget>
      <widget class="QPushButton" name="ImpExpDeck_B">
       <property name="geometry">
        <rect>
         <x>610</x>
         <y>30</y>
         <width>141</width>
         <height>81</height>
        </rect>
       </property>
       <property name="text">
        <string>Import/Export Deck</string>
       </property>
      </widget>
      <widget class="QLabel" name="Icon">
       <property name="geometry">
        <rect>
         <x>1042</x>
         <y>19</y>
         <width>121</width>
         <height>101</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QGroupBox" name="OptionsAndDeck">
      <property name="maximumSize">
       <size>
        <width>280</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="title">
       <string>Extensions</string>
      </property>
      <widget class="QStackedWidget" name="Options">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>0</y>
         <width>261</width>
         <height>281</height>
        </rect>
       </property>
       <property name="currentIndex">
        <number>6</number>
       </property>
       <widget class="QWidget" name="BlankPage"/>
       <widget class="QWidget" name="Health">
        <widget class="QLabel" name="title_H">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>20</y>
           <width>100</width>
           <height>30</height>
          </rect>
         </property>
         <property name="text">
          <string>Health</string>
         </property>
        </widget>
        <widget class="QLabel" name="from_H">
         <property name="geometry">
          <rect>
           <x>2</x>
           <y>60</y>
           <width>41</width>
           <height>21</height>
          </rect>
         </property>
         <property name="text">
          <string>from</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="HealthFrom">
         <property name="geometry">
          <rect>
           <x>50</x>
           <y>60</y>
           <width>51</width>
           <height>21</height>
          </rect>
         </property>
        </widget>
        <widget class="QLineEdit" name="HealthTo">
         <property name="geometry">
          <rect>
           <x>130</x>
           <y>60</y>
           <width>51</width>
           <height>21</height>
          </rect>
         </property>
        </widget>
        <widget class="QLabel" name="to_H">
         <property name="geometry">
          <rect>
           <x>110</x>
           <y>60</y>
           <width>21</width>
           <height>21</height>
          </rect>
         </property>
         <property name="text">
          <string>to</string>
         </property>
        </widget>
       </widget>
       <widget class="QWidget" name="Cost">
        <widget class="QLabel" name="title_C">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>20</y>
           <width>101</width>
           <height>30</height>
          </rect>
         </property>
         <property name="text">
          <string>Cost</string>
         </property>
        </widget>
        <widget class="QLabel" name="from_C">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>60</y>
           <width>41</width>
           <height>21</height>
          </rect>
         </property>
         <property name="text">
          <string>from</string>
         </property>
        </widget>
        <widget class="QLabel" name="to_C">
         <property name="geometry">
          <rect>
           <x>100</x>
           <y>60</y>
           <width>21</width>
           <height>21</height>
          </rect>
         </property>
         <property name="text">
          <string>to</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="CostTo">
         <property name="geometry">
          <rect>
           <x>130</x>
           <y>60</y>
           <width>51</width>
           <height>21</height>
          </rect>
         </property>
        </widget>
        <widget class="QLineEdit" name="CostFrom">
         <property name="geometry">
          <rect>
           <x>40</x>
           <y>60</y>
           <width>51</width>
           <height>21</height>
          </rect>
         </property>
        </widget>
       </widget>
       <widget class="QWidget" name="Attack">
        <widget class="QLabel" name="title_A">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>20</y>
           <width>100</width>
           <height>30</height>
          </rect>
         </property>
         <property name="text">
          <string>Attack</string>
         </property>
        </widget>
        <widget class="QLabel" name="from_A">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>60</y>
           <width>41</width>
           <height>21</height>
          </rect>
         </property>
         <property name="text">
          <string>from</string>
         </property>
        </widget>
        <widget class="QLabel" name="to_A">
         <property name="geometry">
          <rect>
           <x>100</x>
           <y>60</y>
           <width>21</width>
           <height>21</height>
          </rect>
         </property>
         <property name="text">
          <string>to</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="AttackTo">
         <property name="geometry">
          <rect>
           <x>130</x>
           <y>60</y>
           <width>51</width>
           <height>21</height>
          </rect>
         </property>
        </widget>
        <widget class="QLineEdit" name="AttackFrom">
         <property name="geometry">
          <rect>
           <x>40</x>
           <y>60</y>
           <width>51</width>
           <height>21</height>
          </rect>
         </property>
        </widget>
       </widget>
       <widget class="QWidget" name="Type">
        <widget class="QLabel" name="title_T">
         <property name="geometry">
          <rect>
           <x>60</x>
           <y>20</y>
           <width>100</width>
           <height>30</height>
          </rect>
         </property>
         <property name="text">
          <string>Type</string>
         </property>
        </widget>
        <widget class="QCheckBox" name="Type_1">
         <property name="geometry">
          <rect>
           <x>60</x>
           <y>60</y>
           <width>92</width>
           <height>23</height>
          </rect>
         </property>
         <property name="text">
          <string>Champion</string>
         </property>
        </widget>
        <widget class="QCheckBox" name="Type_2">
         <property name="geometry">
          <rect>
           <x>60</x>
           <y>90</y>
           <width>92</width>
           <height>23</height>
          </rect>
         </property>
         <property name="text">
          <string>Follower</string>
         </property>
        </widget>
        <widget class="QCheckBox" name="Type_3">
         <property name="geometry">
          <rect>
           <x>60</x>
           <y>120</y>
           <width>92</width>
           <height>23</height>
          </rect>
         </property>
         <property name="text">
          <string>Spell</string>
         </property>
        </widget>
        <widget class="QCheckBox" name="Type_4">
         <property name="geometry">
          <rect>
           <x>60</x>
           <y>150</y>
           <width>92</width>
           <height>23</height>
          </rect>
         </property>
         <property name="text">
          <string>Landmark</string>
         </property>
        </widget>
       </widget>
       <widget class="QWidget" name="Rarity">
        <widget class="QLabel" name="title_Rar">
         <property name="geometry">
          <rect>
           <x>50</x>
           <y>20</y>
           <width>100</width>
           <height>30</height>
          </rect>
         </property>
         <property name="text">
          <string>Rarity</string>
         </property>
        </widget>
        <widget class="QCheckBox" name="Rarity_1">
         <property name="geometry">
          <rect>
           <x>60</x>
           <y>70</y>
           <width>92</width>
           <height>23</height>
          </rect>
         </property>
         <property name="text">
          <string>Common</string>
         </property>
        </widget>
        <widget class="QCheckBox" name="Rarity_2">
         <property name="geometry">
          <rect>
           <x>60</x>
           <y>100</y>
           <width>92</width>
           <height>23</height>
          </rect>
         </property>
         <property name="text">
          <string>Rate</string>
         </property>
        </widget>
        <widget class="QCheckBox" name="Rarity_3">
         <property name="geometry">
          <rect>
           <x>60</x>
           <y>130</y>
           <width>92</width>
           <height>23</height>
          </rect>
         </property>
         <property name="text">
          <string>Epic</string>
         </property>
        </widget>
       </widget>
       <widget class="QWidget" name="InOutDeckCode">
        <widget class="QPlainTextEdit" name="CodeDisplay">
         <property name="geometry">
          <rect>
           <x>-7</x>
           <y>50</y>
           <width>281</width>
           <height>131</height>
          </rect>
         </property>
        </widget>
        <widget class="QPushButton" name="Import_B">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>200</y>
           <width>86</width>
           <height>28</height>
          </rect>
         </property>
         <property name="text">
          <string>Import</string>
         </property>
        </widget>
        <widget class="QPushButton" name="Export_B">
         <property name="geometry">
          <rect>
           <x>170</x>
           <y>200</y>
           <width>86</width>
           <height>28</height>
          </rect>
         </property>
         <property name="text">
          <string>Export</string>
         </property>
        </widget>
       </widget>
       <widget class="QWidget" name="Region">
        <widget class="QLabel" name="title_Reg">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>30</y>
           <width>100</width>
           <height>30</height>
          </rect>
         </property>
         <property name="text">
          <string>Region</string>
         </property>
        </widget>
        <widget class="QCheckBox" name="Region_1">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>60</y>
           <width>121</width>
           <height>23</height>
          </rect>
         </property>
         <property name="text">
          <string>Bandle City</string>
         </property>
        </widget>
        <widget class="QCheckBox" name="Region_2">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>85</y>
           <width>111</width>
           <height>23</height>
          </rect>
         </property>
         <property name="text">
          <string>Bilgewater</string>
         </property>
        </widget>
        <widget class="QCheckBox" name="Region_3">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>110</y>
           <width>92</width>
           <height>23</height>
          </rect>
         </property>
         <property name="text">
          <string>Demacia</string>
         </property>
        </widget>
        <widget class="QCheckBox" name="Region_4">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>135</y>
           <width>101</width>
           <height>23</height>
          </rect>
         </property>
         <property name="text">
          <string>Freljord</string>
         </property>
        </widget>
        <widget class="QCheckBox" name="Region_7">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>185</y>
           <width>92</width>
           <height>23</height>
          </rect>
         </property>
         <property name="text">
          <string>Noxus</string>
         </property>
        </widget>
        <widget class="QCheckBox" name="Region_5">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>160</y>
           <width>92</width>
           <height>23</height>
          </rect>
         </property>
         <property name="text">
          <string>Ixtal</string>
         </property>
        </widget>
        <widget class="QCheckBox" name="Region_11">
         <property name="geometry">
          <rect>
           <x>120</x>
           <y>110</y>
           <width>92</width>
           <height>23</height>
          </rect>
         </property>
         <property name="text">
          <string>Shurima</string>
         </property>
        </widget>
        <widget class="QCheckBox" name="Region_12">
         <property name="geometry">
          <rect>
           <x>120</x>
           <y>135</y>
           <width>92</width>
           <height>23</height>
          </rect>
         </property>
         <property name="text">
          <string>Targon</string>
         </property>
        </widget>
        <widget class="QCheckBox" name="Region_9">
         <property name="geometry">
          <rect>
           <x>120</x>
           <y>60</y>
           <width>92</width>
           <height>23</height>
          </rect>
         </property>
         <property name="text">
          <string>Piltover</string>
         </property>
        </widget>
        <widget class="QCheckBox" name="Region_10">
         <property name="geometry">
          <rect>
           <x>120</x>
           <y>85</y>
           <width>92</width>
           <height>23</height>
          </rect>
         </property>
         <property name="text">
          <string>The void</string>
         </property>
        </widget>
        <widget class="QCheckBox" name="Region_14">
         <property name="geometry">
          <rect>
           <x>120</x>
           <y>185</y>
           <width>92</width>
           <height>23</height>
          </rect>
         </property>
         <property name="text">
          <string>Zaun</string>
         </property>
        </widget>
        <widget class="QCheckBox" name="Region_13">
         <property name="geometry">
          <rect>
           <x>120</x>
           <y>160</y>
           <width>92</width>
           <height>23</height>
          </rect>
         </property>
         <property name="text">
          <string>Ionia</string>
         </property>
        </widget>
        <widget class="QCheckBox" name="Region_8">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>210</y>
           <width>131</width>
           <height>23</height>
          </rect>
         </property>
         <property name="text">
          <string>Shadow isles</string>
         </property>
        </widget>
       </widget>
      </widget>
      <widget class="QPushButton" name="ShowDeck_B">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>550</y>
         <width>101</width>
         <height>33</height>
        </rect>
       </property>
       <property name="text">
        <string>Show Deck</string>
       </property>
      </widget>
      <widget class="QPlainTextEdit" name="DeckDisplay">
       <property name="geometry">
        <rect>
         <x>5</x>
         <y>259</y>
         <width>270</width>
         <height>281</height>
        </rect>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QGroupBox" name="Cards">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>600</height>
       </size>
      </property>
      <property name="title">
       <string>Card</string>
      </property>
      <widget class="QPushButton" name="GoBack_B">
       <property name="geometry">
        <rect>
         <x>320</x>
         <y>540</y>
         <width>61</width>
         <height>41</height>
        </rect>
       </property>
       <property name="text">
        <string>&lt;—</string>
       </property>
      </widget>
      <widget class="QPushButton" name="GoNext_B">
       <property name="geometry">
        <rect>
         <x>510</x>
         <y>540</y>
         <width>61</width>
         <height>41</height>
        </rect>
       </property>
       <property name="text">
        <string>—&gt;</string>
       </property>
      </widget>
      <widget class="QLabel" name="page">
       <property name="geometry">
        <rect>
         <x>390</x>
         <y>545</y>
         <width>41</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>page</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>462</x>
         <y>545</y>
         <width>41</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>of 50</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="NumberOfPage">
       <property name="geometry">
        <rect>
         <x>426</x>
         <y>547</y>
         <width>31</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>1</string>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1223</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <action name="actionxdsd">
   <property name="text">
    <string>xdsd</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
